app/assets/javascripts/services/service/service_data.js

```diff
@@ -0,0 +1,15 @@
+ManageIQ.angular.app.service('serviceDataFactory', ['API', function(API) {
+  var urlBase = '/api/services';
+
+  this.getServiceData = function (id) {
+    if(angular.isDefined(id)) {
+      return API.get(urlBase + '/' + id).then(handleSuccess);
```

Comment:
we could remove that `.then(handleSuccess)` from here too since it's not doing anything..